{{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml

```diff
@@ -81,6 +81,9 @@ stages:
   - oc tag "${SOURCE}/${CI_PROJECT_NAME}:${CI_COMMIT_SHA}"
            "${TARGET}/${CI_PROJECT_NAME}:${CI_COMMIT_SHA}"
 {%- endif %}
+  - for OBSOLETE in $(oc get configmap --sort-by='.metadata.creationTimestamp' -o name
+      -l component=application,environment=${CI_ENVIRONMENT_NAME} | tail -n +5);
```

Comment:
Still missing the `app=${CI_PROJECT_NAME}` selector

Reply:
True. I'm sorry. Must have overlooked that.

In the end I figured that the `app` really corresponds to the value of `${TARGET}`, as of the current implementation in the overlay. This does not correspond even to `${CI_PROJECT_NAME}` for all deployment strategies.

Reply:
I'm kinda lost now as where which label is used. But I don't think it will work now with `$TARGET`.
In the dedicated case, `app: {{ cookiecutter.project_slug }}` is used but in shared `app: {{ cookiecutter.project_slug }}-integration` etc.